lib/FusionInventory/Agent/Task/Inventory/Win32/License.pm

```diff
@@ -58,7 +58,7 @@ sub doInventory {
 
 sub _getWmiSoftwareLicensingProducts {
 
-    my @licences;
+    my (@licences) = @_;
```

Comment:
You should better get an array ref here: `my ($licenses) = @_;`

In that way, the last `return @licenses;` in the method should be deleted.